.travis.yml

```diff
@@ -2,6 +2,16 @@ language: bash
 services: docker
 
 env:
+  - VERSION=7.3-rc VARIANT=stretch/cli
+  - VERSION=7.3-rc VARIANT=stretch/apache
+  - VERSION=7.3-rc VARIANT=stretch/fpm
+  - VERSION=7.3-rc VARIANT=stretch/zts
+  - VERSION=7.3-rc VARIANT=alpine3.7/cli
+  - VERSION=7.3-rc VARIANT=alpine3.7/fpm
+  - VERSION=7.3-rc VARIANT=alpine3.7/zts
+  - VERSION=7.3-rc VARIANT=alpine3.6/cli
```

Comment:
Do we want alpine 3.6 / 3.7 for 7.3 or should we go full 3.8?

Reply:
Yes, let's do only Alpine 3.8 for php:7.3.  The plan is to migrate the others to 3.8 eventually anyway (probably by dropping 3.6).

Reply:
check.